### PR TITLE
Docs: adds modify dashboard settings docs

### PR DESCRIPTION
**Pull request by chri2547**
This PR adds modify dashboard settings content.

### CODE REVIEW

File: docs/sources/dashboards/build-dashboards/modify-dashboard-settings/index.md [hunk 1]
```diff
@@ -5,25 +5,113 @@ title: Modify dashboard settings
...
+     - **Default:** Grafana uses the default selected time zone for the user profile, team, or organization. If no time zone is specified for the user profile, a team the user is a member of, or the organization, then Grafana uses the local browser time.
+     - **Local browser time:** The time zone configured for the viewing user browser is used. This is usually the same time zone as set on the computer.
+     - Standard [ISO 8601 time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), including UTC.
+   - **Auto-refresh:** Customize the options displayed for relative time and the auto-refresh options Entries are comma separated and accept any valid time unit.
+   - **Now delay:** Override the `now` time by entering a time delay. Use this option to accommodate known delays in data aggregation to avoid null values.
+   - **Hide time picker:** Select this option if you do not want Grafana to display the time picker.
+
+## Add an annotation query
+
+An annotation query is a query that xxx. An annotation query provides the following benefits:
```
Comment (chri2547) [inline docs/sources/dashboards/build-dashboards/modify-dashboard-settings/index.md:48]: @torkelo , need some assistance with this intro

File: docs/sources/dashboards/build-dashboards/modify-dashboard-settings/index.md [hunk 2]
```diff
@@ -5,25 +5,113 @@ title: Modify dashboard settings
...
+1. Enter a name and select a data source.
+1. Complete the rest of the form to build a query and annotation.
+
+> **Note:** The **Dashboard settings** page is dynamic and provides a unique set of query and annotation fields for you to populate.
+
+## Add a variable
+
+Add a variable when you want to create interactive and dynamic dashboards that instantly update when select a variable from a picklist.
+
+What are the benefits here? That I don't need to create 40 different dashboards each with a hard-coded value? More efficient use of dashboard space?
```
Comment (chri2547) [inline docs/sources/dashboards/build-dashboards/modify-dashboard-settings/index.md:71]: @torkelo , some benefits here?

File: docs/sources/dashboards/build-dashboards/modify-dashboard-settings/index.md [hunk 3]
```diff
@@ -5,25 +5,113 @@ title: Modify dashboard settings
...
+1. Enter title and and in the **Type** field, select **Dashboard** or **Link**.
+1. To add a dashboard link:
+
+   a. Add an optional tag.
+
+   Tags are useful for filtering and sorting dashboards.
+
+   b. Select any of the dashboard link **Options**.
+
+   Is there anything helpful we want to say here about any of these options? Only add content here if it's not obvious what any option does.
```
Comment (chri2547) [inline docs/sources/dashboards/build-dashboards/modify-dashboard-settings/index.md:97]: @torkelo , question here for you.

File: docs/sources/dashboards/build-dashboards/modify-dashboard-settings/index.md [hunk 4]
```diff
@@ -5,25 +5,113 @@ title: Modify dashboard settings
...
+
+1. To add a link:
+
+   a. Add a URL and tooltip text that appears when the user hovers over the link.
+
+   b. Select an icon that appears next to the link.
+
+   c. Select any of the dashboard link **Options**.
+
+   We can repeat guidance from above if there are any options that are not obvious to the user.
```
Comment (chri2547) [inline docs/sources/dashboards/build-dashboards/modify-dashboard-settings/index.md:109]: @torkelo , I'll add some of the guidance from above to this section.

File: docs/sources/dashboards/build-dashboards/modify-dashboard-settings/index.md [hunk 5]
```diff
@@ -5,25 +5,113 @@ title: Modify dashboard settings
...
 # Modify dashboard settings
 
-xxxxx.
+The dashboard settings page enables you to:
+
+- edit general dashboard properties, including time settings
+- add annotation queries
+- add dashboard variables
+- add links
+- view the dashboard JSON model
```
Comment (torkelo) [inline docs/sources/dashboards/build-dashboards/modify-dashboard-settings/index.md:23]: should these be capitalized ?

**Review by torkelo — APPROVED** [review]

**Pull request merged**